src/client/renderer/r_image.c

```diff
@@ -75,6 +75,8 @@ static void R_TextureMode(void) {
 typedef struct {
 	uint32_t width;
 	uint32_t height;
+	uint32_t using_rgb;
+	uint32_t using_hdr;
```

Comment:
Fair enough

Reply:
I don't like it when cvars use the imperative form because I always forget which verb (show/draw/use/enable/...) is used, and if it's in front or behind, etc. Also find it easier for autocomplete lists to sort by I-want-all-settings-for-this-component rather than I-want-to-enable-something-and-maybe-you-have-it. *shrug*

Reply:
That's a very fair point and we could probably act on it with a bunch of our cvars. I agree. But this one in particular is not one that the user should mess with. It's CVAR_NO_SET, and he's really only using a cvar as a means to inject branching logic into the shader. That said, I won't push on this rename any further.